lib/main.js

```diff
@@ -22,6 +22,10 @@ module.exports = {
     this.autocompleteManager = null
   },
 
+  provideWatchEditor () {
+    return this.autocompleteManager.watchEditor
```

Comment:
It was prebound before in `autocomplete-manager.js`: `this.watchEditor = this.watchEditor.bind(this)`. I removed it so it's more clear locally.